travis/install-common-deps.sh

```diff
@@ -19,14 +19,14 @@ conda create -q -n test-environment python=3.6
 source activate test-environment
 
 python --version
-pip install --upgrade pip==19.3.1
+pip install --upgrade pip
```

Comment:
Written in to get flaky dependency installation to work, to verify windows tests would pass. This should be factored out into another PR, if it is indeed the fix.